textpipe/doc.py

```diff
@@ -444,7 +444,7 @@ def generate_word_vectors(self, model_name=None):
         >>> doc.word_vectors['Test']['is_oov']
         True
         >>> len(doc.word_vectors['Test']['vector'])
-        384
+        96
```

Comment:
How does this happen?

Reply:
They scaled down the word vectors for speed:
> spaCy v2.1's token vectors are 96 elements wide

https://explosion.ai/blog/spacy-v2-1